bin/installInfo.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+'use strict';
+console.log('##########################################');
+console.log('# Please add the following typings manually:');
+console.log('# typings install dt~es6-collections dt~es6-promise dt~backbone dt~cordova/plugins/device dt~phonegap-plugin-push dt~websql dt~backbone-global dt~q dt~jquery dt~underscore dt~node --save --global');
+console.log('# typings install npm~lodash --save')
```

Comment:
The need for manual installation of typings should arise for global dependencies only. Doesn't it work without explicit installation of these? If not, something is wrong with the SDK setup itself I believe and we need to further investigate this. In this case, we should raise an issue and accept your MR as a workaround.